crates/red_knot_python_semantic/resources/mdtest/binary/integers.md

```diff
@@ -37,6 +37,7 @@ b = 2 // 0  # error: "Cannot floor divide object of type `Literal[2]` by zero"
 c = 3 % 0  # error: "Cannot reduce object of type `Literal[3]` modulo zero"
 d = int() / 0  # error: "Cannot divide object of type `int` by zero"
 e = 1.0 / 0  # error: "Cannot divide object of type `float` by zero"
+f = 1 / False  # error: "Cannot divide object of type `Literal[1]` by zero"
```

Comment:
This tests dividing by `False`, which is good, but you also added a line of code below which supports issuing division by zero diagnostic when dividing a bool by zero (i.e. the dividend is a `BooleanLiteral`). Can we add a test for that, too?

And actually also a test for where the dividend is just `bool()`, too!